src/cljs/main/broadfcui/components/ontology_autosuggest.cljs

```diff
@@ -1,37 +1,64 @@
 (ns broadfcui.components.ontology-autosuggest
   (:require
    [dmohs.react :as react]
+   [clojure.string :as string]
+   [broadfcui.common.flex-utils :as flex]
+   [broadfcui.common.icons :as icons]
+   [broadfcui.common.links :as links]
+   [broadfcui.common.style :as style]
    [broadfcui.components.autosuggest :refer [Autosuggest]]
    [broadfcui.utils :as utils]
    ))
 
+(defn render-doid [doid]
+  (-> doid
+      (string/split "/")
+      last
+      (string/replace-all #"_" ": ")))
```

Comment:
careful of branching/rebasing here. Sometime after you branched (I assume), we removed the space after the colon, so `render-doid` looks like the ids at http://disease-ontology.org/. See the `render-doid` removed from research_purpose.cljs.

Reply:
thanks! good catch